deepchem/models/openvino_model.py

```diff
@@ -0,0 +1,206 @@
+import os
+import io
+import sys
+import itertools
+import subprocess
+
+import numpy as np
+
+import tensorflow as tf
+from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
+
+try:
+  try:
+    # Import for pip package
+    from model_optimizer import mo_tf
+  except:
+    # Import for OpenVINO distribution
+    import mo_tf
+  from openvino.inference_engine import IECore, StatusCode
+  is_available = True
+except:
+  is_available = False
+  pass
+"""
```

Comment:
Can you make this docstring the class docstring for `OpenVINOModel`? That way the autodoc can pick it up